test/integration/targets/unarchive/tasks/test_different_language_var.yml

```
@@ -0,0 +1,40 @@
+- name: test non-ascii with different LANGUAGE
+  block:
```

You can add a conditional here so we only run tasks in this block if apt is supported.
```suggestion
  when: ansible_os_family == 'Debian'
  block:
```